llvm/lib/CodeGen/MachineSink.cpp

```diff
@@ -26,6 +26,7 @@
 #include "llvm/Analysis/AliasAnalysis.h"
 #include "llvm/Analysis/CFG.h"
 #include "llvm/Analysis/ProfileSummaryInfo.h"
+#include "llvm/CodeGen/DeadMachineInstructionElim.h"
```

Comment:
This probably shouldn't depend on another pass's header. Is there any reason why we don't have the full version of DeadMachineInstructionElim's version of isDead directly in MachineInstr instead of in the pass?